src/main/java/com/bobocode/bibernate/QueryHelper.java

```diff
@@ -0,0 +1,36 @@
+package com.bobocode.bibernate;
+
+import com.bobocode.bibernate.exception.QueryHelperException;
+import com.bobocode.bibernate.session.Session;
+import com.bobocode.bibernate.session.SessionFactory;
+
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+public class QueryHelper {
+
+    private QueryHelper() {
+    }
+
+    public static void runWithinTx(SessionFactory sessionFactory, Consumer<Session> action){
+        runWithinTxReturning(sessionFactory, session -> {
+            action.accept(session);
+            return null;
```

Comment:
Just a tip: It doesn't look very good from a reading perspective.As for me, I would rather duplicate the try catch block here.